src/pages/starter-pack/components/from-breeder/from-breeder.module.scss

```diff
@@ -39,14 +40,9 @@
       margin: 3px 0;
     }
   }
-
-  &__another {
-    display: flex;
-    flex-direction: column;
-    margin: 15px 0;
-
-    span {
-      margin: 3px 0;
-    }
-  }
 }
+@media (min-width: 768px) {
+  .fromBreeder{
+    padding-top: 20px;
+  }
+}
```

Comment:
nit: Don't forget to add empty line to the end of the file

Reply:
+